.github/workflows/continuous-integration.yml

```diff
@@ -153,6 +153,15 @@ jobs:
           - locked
```

Comment:
Makes little sense now that we dynamically change major versions of our dependencies

Reply:
so what needs to be done? Just delete "locked"?

Reply:
jup